validator/accounts/wallet_create.go

```diff
@@ -35,6 +35,9 @@ func CreateAndSaveWalletCli(cliCtx *cli.Context) (*wallet.Wallet, error) {
 	if err != nil {
 		return nil, err
 	}
+	if keymanagerKind == keymanager.Imported {
+		return nil, errors.New("could not initialize wallet for imported accounts. Please use `validator accounts import` instead")
```

Comment:
```suggestion
		return nil, errors.New("could not initialize wallet for imported accounts - please use `validator accounts import` instead")
```
We don't normally use capital letters in error messages, so I think this will be more consistent with the rest of the codebase.